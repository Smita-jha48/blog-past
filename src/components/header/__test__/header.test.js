import { render } from '@testing-library/react';
import Header from '../header';

describe('Header', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
