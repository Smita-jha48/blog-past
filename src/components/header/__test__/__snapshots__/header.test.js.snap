// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<DocumentFragment>
  <nav
    class="navbar"
  >
    <div
      class="logo"
    >
      <div
        class="titleUpper"
      >
        The Artifact
      </div>
      <div
        class="titleLower"
      >
        Culture & Art blog
      </div>
    </div>
    <ul
      class="nav-links"
    >
      <div
        class="menu"
      >
        <li>
          <a
            href="/"
          >
            Blog
          </a>
        </li>
        <li>
          <a
            href="/"
          >
            About
          </a>
        </li>
        <li>
          <a
            href="/"
          >
            Contact
          </a>
        </li>
      </div>
    </ul>
  </nav>
</DocumentFragment>
`;
